From cdd6e10addb4d19ff709f91f40c7a474c63f1c50 Mon Sep 17 00:00:00 2001
From: Jaroslav Kysela <jkysela@redhat.com>
Date: Wed, 3 Sep 2014 07:31:01 -0400
Subject: [alsa] hda/realtek: Add two codecs alias name for Dell

Message-id: <140972823573-277-patchset-jkysela@redhat.com>
Patchwork-id: 90973
O-Subject: [RHEL 7.1 PATCH 276/442] ALSA: hda/realtek - Add two codecs alias name for Dell
Bugzilla: 1112200
RH-Acked-by: John Linville <linville@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Ivan Vecera <ivecera@redhat.com>

Add ALC3235 ALC3263.

Signed-off-by: Kailang Yang <kailang@realtek.com>
Signed-off-by: Takashi Iwai <tiwai@suse.de>
Signed-off-by: Jaroslav Kysela <perex@perex.cz>

(picked from kernel: 193177de4ff8b7b8a654b198611ba774c345f15c)
rhbz#1112200
Signed-off-by: Jaroslav Kysela <jkysela@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/sound/pci/hda/patch_realtek.c b/sound/pci/hda/patch_realtek.c
index 5994ded..6a02386 100644
--- a/sound/pci/hda/patch_realtek.c
+++ b/sound/pci/hda/patch_realtek.c
@@ -951,7 +951,9 @@ static struct alc_codec_rename_pci_table rename_pci_tbl[] = {
  { 0x10ec0280, 0x1028, 0, "ALC3220" },
  { 0x10ec0282, 0x1028, 0, "ALC3221" },
  { 0x10ec0283, 0x1028, 0, "ALC3223" },
+ { 0x10ec0288, 0x1028, 0, "ALC3263" },
  { 0x10ec0292, 0x1028, 0, "ALC3226" },
+ { 0x10ec0293, 0x1028, 0, "ALC3235" },
  { 0x10ec0255, 0x1028, 0, "ALC3234" },
  { 0x10ec0668, 0x1028, 0, "ALC3661" },
  { } /* terminator */
-- 
1.7.1